config/load.go

```diff
@@ -39,6 +40,15 @@ func Load(logger *zerolog.Logger) (appConfig *AppConfig, err error) {
 		return nil, err
 	}
 
+	if appConfig.Debug {
+		cfg, err := json.MarshalIndent(appConfig, "", "  ")
+		if err != nil {
+			logger.Error().Msg("Unable to decode App Config to json")
+		} else {
+			fmt.Printf("loaded config: %s", cfg)
+		}
```

Comment:
1. as we have a struct with config, why don't you just use `Printf("loaded config: %v", appConfig)`?
2. Why you're doing fmt.Printf instead of using a logger?

Reply:
1. Because it will print it in one-liner.
2. Because it will print it in one-liner 😄

Doing it this way, we print it as a formatted json which makes it MUCH easier to look at quickly and seeing if everything is okay. This is also shown only when when `debug` flag is selected. That's why I decided to make an exception here and use the standard fmt.Printf function.